.vsts-ci/releasePhase.yml

```diff
@@ -99,15 +104,38 @@ jobs:
     displayName: 'docker login $(dockerHost)'
     condition: and( succeededOrFailed(), ne(variables['Channel'],''))
 
-  - pwsh: ' az login --service-principal -u $(az_url) -p $(az_key) --tenant $(az_name)'
+    
+  - pwsh: |
+      if($env:ACR -eq 'yes')
+      {
+        Write-Host 'Using ACR, will need to az login'
+        az login --service-principal -u $(az_url) -p $(az_key) --tenant $(az_name)
+      }
     displayName: 'az login'
     condition: and( succeededOrFailed(), ne(variables['Channel'],''))
 
-  - pwsh: |
-      az account set --subscription $(StorageSubscriptionName)
-      $querystring = az storage account generate-sas --account-name $(StorageAccount) --services b --resource-types o --expiry ((get-date).AddDays(1).ToString("yyyy-MM-dd")) --permission r --https-only | convertfrom-json
-      $url = "https://$(StorageAccount).blob.core.windows.net/?$querystring"
-      Write-Host "##vso[task.setvariable variable=SasUrl;]$url"
+  - task: AzurePowerShell@5
+    inputs:
+      azureSubscription: '$(AzureFileCopySubscription)'
+      scriptType: inlineScript
+      azurePowerShellVersion: latestVersion
+      pwsh: true
+      inline: |
+        $modules = 'Az.Accounts'
+        foreach ($module in $modules) {
+          $modulesOnMachine = get-module -listavailable
+          Write-Host $modulesOnMachine -Verbose
+          if (!(Get-Module $module -listavailable)) {
+            Write-Verbose "installing $module..." -verbose
+            Install-Module $module -force -allowclobber
+          } else {
+            Write-Verbose "$module is already installed" -verbose
+          }
+        }
```

Comment:
This needs to go into a separate pwsh task before you ever trying to use Azure PowerShell.

Explanation:  You can't user AzurePowerShell if it is not installed yet.